src/scripts/python/filter_by_num_callers.py

```diff
@@ -24,7 +24,9 @@
                 ref = data.group(1)
                 var = data.group(2)
                 callers = data.group(3).split(",")
-                if len(callers) > 1:
```

Comment:
Patrik, can you look especially at this part to see that it seems correct!

Reply:
Looks good! But we should maybe in the future consider adding possibility to of add the variant to a class (like high support, low support) instead of removing the variants.